src/main/java/org/jenkinsci/test/acceptance/plugins/ssh_slaves/SshSlaveLauncher.java

```diff
@@ -44,6 +44,11 @@ public SshCredentialDialog addCredential() {
         return new SshCredentialDialog(getPage(), "/credentials");
     }
 
+    public void setJavaPath(String jvmPath) {
+        try (UselessFileDetectorReplacement ufd = new UselessFileDetectorReplacement(driver)) {
```

Comment:
if anything else uses `path`ish like Strings they would need to be similarly wrapped, however no other code has been observed to fail yet.